packages/grafana-runtime/src/measurement/perf.ts

```diff
@@ -0,0 +1,27 @@
+let lastUpdate = Date.now();
+
+if (!(window as any).grafanaStreamingPerfBudget) {
+  (window as any).grafanaStreamingPerfBudget = 1.05;
```

Comment:
good idea?

This lets us experiment with different values easily in a production build..

![image](https://user-images.githubusercontent.com/705951/113459264-ce131a00-93c9-11eb-9967-d29c844ddace.png)

Reply:
ok for temp, not so sure about permanent. does it have to be on window or is there already some window._gfConfig exposed that we can go in one level. reading from globals in a perf sensitive loop makes me nervous.